source/_components/device_tracker.tile.markdown

```diff
@@ -25,6 +25,10 @@ device_tracker:
   - platform: tile
     username: email@address.com
     password: MY_PASSWORD_123
+    monitored_variables:
```

Comment:
We try to keep all configuration samples minimal. Thus, no optional requirement in the default sample. This helps users to get started quickly by copy-&-paste the sample without worrying about optional parameters which they most likely not need. If required, insert a full configuration sample later that covers special setups or alike.